src/segno-ui.less

```diff
@@ -1,32 +1,5 @@
-// CONFIG
```

Comment:
This file only handles default styling for custom elements, before they are upgraded (preventing FOUC-like behavior).